.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "build/openapi-extractor"]
```

Comment:
New git submodule and things.

@blizzz can you dive the docker thing from the builder a test run to make sure it doesn't break the next RC docs?

Reply:
> New git submodule and things.
> 
> @blizzz can you dive the docker thing from the builder a test run to make sure it doesn't break the next RC docs?

at least building the documentation does not fail and the output structured is as expected (as before). The developer manual is not being distributed. So, all is looking fine.